tests/test_templates.py

```diff
@@ -22,12 +22,12 @@
 * When adding a new interface, try to import it and extend the fixture ``interfaces``. Also add the interface
   gradient computation to the TestGradientIntegration tests.
 
-* When adding a new template, extend the fixtures ``QUBIT_CONSTANT_INPUT`` or ``CV_CONSTANT_INPUT``
+* When adding a new template, extend the fixtures ``QUBIT_DIFFABLE_NONDIFFABLE`` or ``CV_DIFFABLE_NONDIFFABLE``
   by a *list* of arguments to the
   template. Note: Even if the template takes only one argument, it has to be wrapped in a list (i.e. [weights]).
```

Comment:
These second sentences also need to be updated right? I.e., since we're now passing a dict

Reply:
Good find